backend/Lingva.WebAPI/Controllers/MembershipController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using System.IdentityModel.Tokens.Jwt;
+using System.Text;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authorization;
+using UsersManagementBackEnd.Services;
+using Lingva.DataAccessLayer.Entities;
+namespace UsersManagementBackEnd.Controllers
+{
+    [Route("api/[controller]")]
```

Comment:
Add here [ApiController] and extend class from ControlerBase.
Case we don't use View system.